shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-suite/src/test/resources/env/tbl/init-sql/h2/init.sql

```diff
@@ -78,6 +83,8 @@ CREATE TABLE t_order_9 (order_id INT NOT NULL, user_id INT NOT NULL, status VARC
 CREATE TABLE t_order_item_9 (item_id INT NOT NULL, order_id INT NOT NULL, user_id INT NOT NULL, status VARCHAR(45) NULL, creation_date DATE, PRIMARY KEY (item_id));
 CREATE INDEX order_index_t_order_9 ON t_order_9 (order_id);
 
+CREATE TABLE t_order_federate (order_id INT NOT NULL, user_id INT NOT NULL, status VARCHAR(45) NULL, PRIMARY KEY (order_id));
```

Comment:
@LeeGuoPing t_order_federate is a single table, please move it to `h2/init-db_0.sql`